src/main/java/com/google/alpollo/AuthenticationServlet.java

```diff
@@ -1,16 +1,35 @@
 package com.google.alpollo;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
-import com.google.appengine.api.users.UserService;
+import com.google.gson.Gson;
 
 @WebServlet("/auth")
 public class AuthenticationServlet extends HttpServlet {
+    private final Gson gson = new Gson();
+
+    private class LoggedInInfo {
+        final boolean loggedIn;
+        final String loginUrl;
```

Comment:
this field can also mean logoutUrl, maybe it's better to name it authUrl (I have no better idea).